kubectl-gadget.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Dockerfile for the kubectl-gadget plugin.
```

Comment:
Why you didn't create a make target to build this container?

Reply:
Added.